### PR TITLE
Playwright component testing

### PR DESCRIPTION
**Pull request by oreqizer**
Adds Playwright component testing tests inside `Component.ct.tsx`

The component to be tested is in a `Component.ct-story.tsx` file bcs of https://playwright.dev/docs/test-components#test-stories
 Storybook: https://orbit-mainframev-oreqizer-try-playwright-ct.surge.sh

### CODE REVIEW

File: packages/orbit-components/playwright-ct.config.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    trace: "on-first-retry",
+    testIdAttribute: "data-test",
+    ctPort: 3100,
+  },
+  projects: [
+    {
+      name: "Small Mobile",
+      use: {
+        ...devices["iPhone SE"],
+        viewport: { ...devices["iPhone SE"].viewport, width: 320 },
```
Comment (mainframev) [inline packages/orbit-components/playwright-ct.config.ts:25]: I think you can use token values here for width, so it's not hardcoded :)

**Comment by mainframev** [comment]
@oreqizer could you also add `.ct` files to build ignore and type declaration build ignore?  es and lib folders include those files, I do not think they should be a part of the output :) 

Add them [here](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/config/build/consts.mts) and check output ;)

**Review by mainframev — CHANGES_REQUESTED** [review]

File: packages/orbit-components/.gitignore [hunk 2]
```diff
@@ -18,3 +18,7 @@ orbit-icons-font.zip
 orbit-svgs.zip
 cypress/screenshots
 cypress/videos
+node_modules/
```
[the remark this answers is not in the document]
Comment (oreqizer) [reply packages/orbit-components/.gitignore]: cleaned up

File: packages/orbit-components/playwright-ct.config.ts (hunk 1 above)
Comment (oreqizer) [reply packages/orbit-components/playwright-ct.config.ts]: adjusted

File: packages/orbit-components/config/build/consts.mts [hunk 3]
```diff
@@ -14,7 +14,8 @@ export const COMPILE_IGNORE_PATTERNS = [
   // TODO: remove that after TW migration, tailwind is temporary folder
   "**/tmp-tailwind/**/*.{ts?(x),d.ts}",
   "**/*.d.ts",
-  "**/*.stories.*",
```
Comment (mainframev) [inline packages/orbit-components/config/build/consts.mts:17]: this shouldn't be removed, we still have storybook and stories shouldn't be in the builded output 🙂

File: packages/orbit-components/config/build/consts.mts [hunk 4]
```diff
@@ -14,7 +14,8 @@ export const COMPILE_IGNORE_PATTERNS = [
...
+  "**/*.ct.*",
+  "**/*.ct-story.*",
```
Comment (mainframev) [inline packages/orbit-components/config/build/consts.mts:18]: 🙏🏻 add also to ignore for type declarations [here](https://github.com/kiwicom/orbit/blob/master/packages/orbit-components/config/build/consts.mts#L24)

File: packages/orbit-components/config/build/consts.mts (hunk 3 above)
Comment (oreqizer) [reply packages/orbit-components/config/build/consts.mts:17]: ah lol don't know why it was removed

File: packages/orbit-components/config/build/consts.mts (hunk 4 above)
Comment (oreqizer) [reply packages/orbit-components/config/build/consts.mts:19]: fixed

**Review by mainframev — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**